src/lib/Component/EventSubscriber/TwigComponentCollectorSubscriber.php

```diff
@@ -37,6 +37,6 @@ public function onRenderGroup(RenderGroupEvent $event): void
 
     public function onRenderSingle(RenderSingleEvent $event): void
     {
-        $this->collector->addRenderedComponent($event->getGroupName(), $event->getName());
+        $this->collector->addRenderedComponent($event->getGroupName(), $event->getName(), $event->getComponent());
```

Comment:
I wonder if we shouldn't just pass `$event` here, since all the parameters can be easily fetched from it in the collector itself.

Reply:
You are tiying rendering/collecting with event namespace tho, not sure if fan of it.

Reply:
Yeah, you might be right about this. This one needs reverting @wiewiurdp, sorry for the hassle. 😅